### ISSUE TITLE
Add page numbers in the infinite scroll navigation

### ISSUE THREAD

**Issue by seafoox** [algolia/wordpress-search#5]
There are a lot of discussion around infinite scroll, some people like it, some other don't most of the time due to the lack of pagination. Based on discussion I had with several people in the last months, it seems that a good way / compromise is to offer a good UX is to add page numbers to the results: like for instance on Google image

![image](https://cloud.githubusercontent.com/assets/1101220/8143435/3c963a08-11ac-11e5-859e-a2d7bdd21767.png)

Once that done, we could easily add the page number in the URL, so people could jump straight to the right page. They would then have a button on top of the list to load / display previous pages. 

Since it would be our first demo to implement this best practice, it could be a real good example to share, and in the same time, a good opportunity for us to spread the world.

What do you think?

**Comment by pixelastic** [comment]
Agreed that page numbers are a must have. I like the idea of a shareable url for an infinite scroll but I never saw it anywhere.

**Comment by redox** [comment]
Like the idea; @pixelastic wanna give it a try? I would like to have your feedback on the coding style + dev environment as well :+1:

**Comment by pixelastic** [comment]
I can give it a shot, but I already have a few items pilling up in my to-do list. How urgent /important is this?

**Comment by redox** [comment]
Very minor :+1:

**Comment by redox** [comment]
Actually the more I think about it, the more I think displaying the page number doesn't make sense for infinite scroll.
